connector_jira/models/res_users/common.py

```diff
@@ -127,4 +134,12 @@ def search(self, fragment=None):
         users = self.client.search_users(fragment, maxResults=None,
                                          includeActive=True,
                                          includeInactive=True)
+
+        # User 'key' is unique and if same key appears several times, it means
+        # that same user is present in multiple User Directories
+        users = list(map(
```

Comment:
what about `users = list(set([u.key for u in users]))`?

Reply:
Also an option if we want to pass only key to the caller, though key is something sometimes quite random (especially after cloud-to-server migration)